app/controllers/app.py

```diff
@@ -1519,3 +1519,33 @@ def post(self, project_id, app_id):
             return dict(status='fail', message='No values found'), 404
 
         return dict(status='success', data=dict(storage_capacity=values, storage_percentage_usage=volume_perc_value)), 200
+
+
+class AppDataSummaryView(Resource):
+
+    @admin_required
+    def post(self):
+        """
+        Aggregate apps per month or year
+        """
+        app_filter_data = request.get_json()
+        filter_schema = AppGraphSchema()
+
+        validated_query_data, errors = filter_schema.load(app_filter_data)
+
+        if errors:
+            return dict(status='fail', message=errors), 400
+
+        start = validated_query_data.get('start', '2018-01-01')
+        end = validated_query_data.get('end', datetime.datetime.now())
+        set_by = validated_query_data.get('set_by', 'month')
+        total_apps = len(App.find_all())
+        
+        app_info = App.graph_data(self=App, start=start, end=end, set_by=set_by)
```

Comment:
After its fixed you wont need to add `self` initialisation
```suggestion
        app_info = App.graph_data(start=start, end=end, set_by=set_by)
```